x-pack/plugins/banners/server/ui_settings.ts

```diff
@@ -30,8 +30,7 @@ export const registerSettings = (uiSettings: UiSettingsServiceSetup, config: Ban
         defaultMessage: 'Banner placement',
       }),
       description: i18n.translate('xpack.banners.settings.placement.description', {
-        defaultMessage:
-          'Display a top banner for this space, above the Elastic header. {subscriptionLink}',
+        defaultMessage: 'Display a top banner above the Elastic header. {subscriptionLink}',
```

Comment:
When spaces are enabled, the 'for this space' part of the description is actually very significant, as global banners are configured via a totally different mechanism... But I guess there's not much we can do here.

Reply:
Good to know. My original thought was to switch the text based on whether spaces is enabled, but if there is a separate global banner configuration somewhere, this whole section could just be removed when spaces is disabled.